pytx3/pytx3/cli/fetch2.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import requests
+import json
+import collections
+import pathlib
+import time
+import typing as t
+from .. import TE
+from ..dataset import Dataset
+from ..indicator import ThreatIndicator
+from ..content_type import meta
+from . import command_base
+
+class Fetch2Command(command_base.Command):
```

Comment:
I'm unsure what the final command name should be here, but for now, we could go with something like `ExperimentalFetchCommand` (and change the related cli invocation words to match) to make it clear this is not intended for production use yet.

Reply:
@InderH, I'll leave to you